src/Ilios/CoreBundle/Tests/Controller/UserControllerTest.php

```diff
@@ -287,6 +287,49 @@ public function testPostUser()
     /**
      * @group controllers_b
      */
+    public function testPostMultipleUsers()
+    {
+        $data = [];
+        $data[] = $this->container->get('ilioscore.dataloader.user')->create();
+        $data[] = $this->container->get('ilioscore.dataloader.user')->create();
```

Comment:
seems like kruft, as the `$data` array get's populated in the loop below. please remove this line and the one above.